python-package/lightgbm/engine.py

```diff
@@ -199,6 +199,8 @@ def train(params, train_set, num_boost_round=100,
         callbacks = set()
     else:
         for i, cb in enumerate(callbacks):
+            if hasattr(cb, 'first_metric_only') and cb.first_metric_only and feval is not None:
```

Comment:
`hasattr(cb, 'first_metric_only') and cb.first_metric_only` -> `getattr(cb, 'first_metric_only', False)`